pkg/okteto/preview.go

```diff
@@ -136,6 +143,42 @@ func ListPreviews(ctx context.Context) ([]PreviewEnv, error) {
 	return body.Previews, nil
 }
 
+func ListPreviewsEndpoints(ctx context.Context, previewName string) ([]string, error) {
+	endpoints := make([]string, 0)
+
+	q := fmt.Sprintf(`query{
+		preview(id: "%s"){
+			deployments{
+				endpoints{
+					url
+				}
+			},
+			statefulsets{
+				endpoints{
+					url
+				}
+			}
+		}
+	}`, previewName)
+	var body PreviewBody
+	if err := query(ctx, q, &body); err != nil {
+		return nil, err
+	}
+
+	for _, d := range body.Preview.Deployments {
+		for _, endpoint := range d.Endpoints {
+			endpoints = append(endpoints, endpoint.URL)
```

Comment:
@maroshii how is the Okteto UI sorting the endpoints?
We should follow the same logic here

Reply:
@pchico83  In the UI endpoints endpoints are sorted by length. Shorter endpoints first. I implemented that logic on the deploy preview github action: https://github.com/okteto/deploy-preview/blob/master/message.go#L174

Reply:
@jLopezbarb let's add that logic here too